### DISCUSSION TITLE
filter doesn't work on relational fields

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#10203] [General]
Hi , 

I want to apply filter on grid list with relational fields but its not working on relational fields like if product list have category  relation.

let me know how it will work ?

**Comment by dvesh3** [comment]
upcoming feature https://github.com/pimcore/pimcore/pull/10022 scheduled for 10.2 release

**Reply by shubhangi92** [reply]
Ok, Thank you!

**Reply by shubhangi92** [reply]
@dvesh3 filters and sorting are still not working with pimcore 10.1.2 version for operator object field getter extractor like type of  many to many relation, workflow states,

**Reply by dvesh3** [reply]
@shubhangi92 This is a new feature scheduled for **10.2** minor release and not a candidate for a bugfix release. 

If you are really interested in this feature, then please help us review https://github.com/pimcore/pimcore/pull/10022 ,  so that we can merge it faster 😉

**Reply by shubhangi92** [reply]
@dvesh3 yes i am expecting this feature of  filter  on relational fields  to be  released asap , as its project requirement,  also can  we have filter on workflow states in this version?
